web/apps/photos/src/utils/file/index.ts

```diff
@@ -681,3 +690,11 @@ const fixTimeHelper = async (
 ) => {
     setFixCreationTimeAttributes({ files: selectedFiles });
 };
+
+const setBulkFavorite = async (files: EnteFile[]) => {
+    try {
+        await addMultipleToFavorites(files);
+    } catch (e) {
+        log.error("Could not add to favorites", e);
```

Comment:
This is swallowing the error, and will cause the local state to get out of sync with remote. I'll merge and fix.

As a side note, we might have other (sometimes temporary) reasons why a particular change has been made yet. And with code changes, is also hard for you to test exhaustively (e.g. your PR yesterday was causing the [shared albums to stop loading](https://github.com/ente-io/ente/commit/ac9e9079619fb8968a6ef254b1ddc4d4b13c778d)).

Reply:
Ok thank you for you review and time. I'll check your fix in following commits so I'll learn.
Indeed, hard for me to make exhaustive tests. If your prefer I can tell you beforehand starting a new change so you can advice not to do it, or to pay attention to some point while testing.
Or I can continue like this. Up to you.